app/src/main/java/org/mozilla/fenix/exceptions/login/LoginExceptionsInteractor.kt

```diff
@@ -0,0 +1,35 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.fenix.exceptions.login
+
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.launch
+import mozilla.components.feature.logins.exceptions.LoginException
+import mozilla.components.feature.logins.exceptions.LoginExceptionStorage
+import org.mozilla.fenix.exceptions.ExceptionsInteractor
+
+/**
+ * Interface for the ExceptionsViewInteractor. This interface is implemented by objects that want
+ * to respond to user interaction on the ExceptionsView
+ */
+interface LoginExceptionsInteractor : ExceptionsInteractor<LoginException>
```

Comment:
nit should this kdoc be on the ExceptionsInteractor interface? Seems more general 🤔